docker/zarr-to-loom/optimus_zarr_to_loom.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+import zarr
+import loompy
+import scipy as sc
+from scipy.sparse import coo_matrix
+import argparse
+
+
+# Custom Exception
+class UnexpectedInputFormat(Exception):
+    pass
+
+
+def main():
+    # Parse the arguments
+    description = """This script converts ZARR optimus output to loom format"""
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('--input-zarr', dest="input_zarr_path", required=True, help="Path to input ZARR file")
+    parser.add_argument('--output-loom', dest="output_loom_path", required=True, help="Path to output loom file")
+    parser.add_argument('--sample_id', dest="sample_id", required=True, help="Sample identifier")
+    args = parser.parse_args()
+
+    input_zarr_path = args.input_zarr_path
+    output_loom_path = args.output_loom_path
+    sample_id = args.sample_id
+
+    # Checks on inputs
+    if not os.path.isdir(input_zarr_path):
+        sys.exit("Error: the input ZARR path is not a directory.")
+    if os.path.exists(output_loom_path):
+        sys.exit("Error: The output loom file exists!")
+
+    # Open the ZARR
+    store = zarr.DirectoryStore(input_zarr_path)
+    root = zarr.open(store)
+
+    # Get the expression matrix
+    # expression matrix in numpy ndarray format (dense)
+    # NOTE: If memory is limiting this could be done by chunk
+    expr = root[f"/{sample_id}.zarr/expression"][:]
```

Comment:
`/{sample_id}.zarr/` is repeated many times in the script. Might be worth it to pull it out as a constant.